admin/release.py

```diff
@@ -303,3 +319,178 @@ def publish_docs_main(args, base_path, top_level):
         sys.stderr.write("%s: Can't publish non-tagged version to production."
                          % (base_path.basename(),))
         raise SystemExit(1)
+
+
+class UploadOptions(Options):
+    """
+    Options for uploading packages.
+    """
+    optParameters = [
+        ["target", None, b'clusterhq-yum-repository',
```

Comment:
I think this should be something more generic. I think we will be uploading DEBs here as well, and perhaps also python sdists and/or wheels.